steps/cakebuild/1.0.0/step.yml

```diff
@@ -0,0 +1,66 @@
+title: Execute Cake C# Script
+summary: Cake (C# Make) is a cross platform build automation system
+description: "Cake (C# Make) is a cross platform build automation system with a C#
+  DSL to do things like compiling code, \ncopy files/folders, running unit tests,
+  compress files and build NuGet packages."
+website: http://cakebuild.net/
```

Comment:
`website` should point to the step's website (most of the times, it's github url), please mention the Cake's website in step's `description` instead.